### DISCUSSION TITLE
Windows 11 installation

### DISCUSSION THREAD

**Discussion by Grill1** [community/community#31086] [Other Feature Feedback, Questions, & Ideas]
Hello had successfully installed win 11 using rufus version 3.16 but wanted to reinstall win 11 again (pc running slow, hotter) 
Tried with 3.18 and 3.20 but dropdown does not show the bypass option any longer, only has 2 options standard installation and  windows to go, but the bypass option is gone now.  Has something changed with the newer rufus versions? Unfortunately had to install win 10, against my wishes. I found this same question but could not find an answer to it. Sorry if I missed it. I really want to keep using win 11, if that’s still possible to accomplish with rufus (in order to bypass all win 11 checks/requirements)?  Thanks very much

**Comment by jfullstackdev** [comment]
is this the project you are referring to ? 

https://github.com/pbatard/rufus

you can also raise an issue there to ask the same question

**Comment by Grill1** [comment]
Thanks very much for reply, had to step away, will pick up again soon… thanks much



On Sunday, August 28, 2022, 11:05 AM, jdevstatic ***@***.***> wrote:




is this the project you are referring to ?

https://github.com/pbatard/rufus

you can also raise an issue there to ask the same question
